### PR TITLE
Pass formData parameters as keyword arguments

### PR DESCRIPTION
**Pull request by jvgutierrez**
As already commented in https://github.com/zalando/connexion/issues/132 this PR adds support to pass formData parameters as keyword parameters to function handlers.

### CODE REVIEW

**Comment by hjacobs** [comment]
@jvgutierrez thanks for the PR! Could you resolve the conflicts please?

**Comment by jvgutierrez** [comment]
@hjacobs done :)

File: tests/test_app.py [hunk 1]
```diff
@@ -576,6 +576,21 @@ def test_falsy_param(app):
     response = json.loads(resp.data.decode())
     assert response == 1
 
+
+def test_formData_param(app):
+    app_client = app.app.test_client()
+    resp = app_client.post('/v1.0/test-formData-param', data=dict(formData='test'))
```
Comment (jmcs) [inline tests/test_app.py]: Thanks for the PR. Can you just change the dict here and on line 590 to the literal form (`{formData: ....}`) for consistency with the rest of the tests?

Comment (jvgutierrez) [inline tests/test_app.py]: Done :)

**Comment by hjacobs** [comment]
:+1: LGTM

**Pull request merged**